src/core/jit/model.rs

```diff
@@ -1,6 +1,7 @@
 use std::collections::HashMap;
 use std::fmt::{Debug, Formatter};
 
+use async_graphql_value::ConstValue;
```

Comment:
Model shouldn't include concrete types. Keep the model parametric for maximum flexibility.

Reply:
not my changes, the usage of ConstValue is came from skip/include pr. I just replaced it with import